setup.py

```diff
@@ -46,8 +46,8 @@
     ],
     python_requires='>=3.7',
     install_requires=[
-        'setuptools',
```

Comment:
`setuptools` is still needed as long as we are using `pkg_resources` to create the namespaces, see https://github.com/zopefoundation/zope.sqlalchemy/blob/master/src/zope/__init__.py